css-flexbox/css-flexbox/align-items.html

```diff
@@ -0,0 +1,30 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <meta http-equiv="X-UA-Compatible" content="ie=edge" />
+    <title>CSS Flexbox - align-items</title>
+    <link rel="stylesheet" href="reset.css" />
+    <link rel="stylesheet" href="pokemon.css" />
+    <style>
+      /* your styles here 👇 */
+      #card-deck {
+        display: flex;
+        min-height: 100vh;
+        align-items: center;
+        justify-content: center;
+        flex-basis: calc(100% / 3);
```

Comment:
`flex-basis` should be applied to the `.card-wrapper` class

Reply:
![Capture](https://github.com/aehrenkranz/m1-0723-code-solutions/assets/130402253/bae554c2-3856-483a-9d6a-5cad7443f805)

Updated to apply flex-basis value to .card-wrapper class.